packages/clay-css/src/content/form_custom.html

```diff
@@ -252,6 +252,35 @@ <h3>Custom File</h3>
 	</div>
 </div>
 
+<div class="clay-site-row-spacer row">
+	<div class="col-md-12">
+		<h3>Custom Range</h3>
+
+		<div class="alert alert-warning">
+			This component needs the prefixes: <code>-webkit-appearance: none; -moz-appearance: none;</code>
+		</div>
+
+		<div class="sheet">
+			<label for="customRange">Custom range example</label>
+			<input class="custom-range" id="customRange" type="range" min="0" max="100"/>
+		</div>
+
+		<h3>Custom Range with blue progress</h3>
+
+		<div class="alert alert-warning">
+			According to the design, progress should be colored blue, to do this there is an <code>::after</code> without with, the idea is adding the width by js reading the position of marker
+		</div>
+
+		<div class="sheet">
+			<label for="customRange">Custom range example</label>
+			<div class="custom-range-progress">
+				<input class="custom-range" id="customRange" type="range" min="0" max="100"/>
```

Comment:
Hey @matuzalemsteles, @marcoscv-work is on PTO today. He'll be back temporarily tomorrow, but it'd be better if you can take it from here to avoid delays :)

Reply:
BTW, I assume this should be as simple as adding the input with the `disabled` attribute. You can see the default BS4 component at https://getbootstrap.com/docs/4.1/components/forms/#range-inputs

Reply:
Oh thanks for the information @jbalsas. Yeah I thought about it but it doesn't look like the `disabled` state of the Lexicon document, some styles are missing.

Reply:
@jbalsas @bryceosterhaus @matuzalemsteles I have a new range input I'm almost done with that we can customize more than Bootstrap's version. The tooltip can be included via html/css.